### ISSUE TITLE
How to train to the precision in the paper

### ISSUE THREAD

**Issue by xudaopao** [facebookresearch/ic_gan#34]
Hi, I'm interested in this work, but I can't reproduce the whole training yet, even if I follow the whole process. Please advise if there are some tips I don't know. Thank you very much for your generosity

**Comment by ArantxaCasanova** [comment]
Hello @xudaopao,

I would need specific details of what exactly you cannot reproduce so that I can provide you with specific help.

As a general tip, note that the FID numbers reported in the paper are computed following this [procedure](https://github.com/facebookresearch/ic_gan#how-to-test-the-models) and that they may be different than the FID scores that are monitored during training.

**Comment by xudaopao** [comment]
Thank you very much for your reply.
The environment I am using is configured according to the provided documentation. But I can never train to reach the target accuracy, even if I use the same configuration file. I also followed the test notes
For example coco-stuff,128*128 big-gan frame.
Thank you again for your generous reply!

**Comment by xudaopao** [comment]
> 您好@xudaopao，
> 
> 我需要具体的细节，说明你到底不能复制什么，以便我能为你提供具体的帮助。
> 
> 作为一般提示，请注意，论文中报告的 FID 数字是按照[此过程](https://github.com/facebookresearch/ic_gan#how-to-test-the-models)计算的，它们可能与训练期间监控的 FID 分数不同。

I am still experimenting at the moment and look forward to your reply

**Comment by xudaopao** [comment]
My main confusion is . Does the config in the file, such as the learning rate and other hyperparameters, need additional tuning?

**Comment by ArantxaCasanova** [comment]
Hello @xudaopao, 
no, they do not need additional tuning. The hyper-parameters in the config file are the ones used in the paper and results should be reproduced by training with them.

Which config file, resolution and dataset are you using? Also, are you training for enough epochs? We use early stopping in our experiments.

**Comment by xudaopao** [comment]
> Hello @xudaopao, no, they do not need additional tuning. The hyper-parameters in the config file are the ones used in the paper and results should be reproduced by training with them.
> 
> Which config file, resolution and dataset are you using? Also, are you training for enough epochs? We use early stopping in our experiments.

Thank you for your generous reply!
I will continue to follow up this interesting job.
By adding training batches. Things are getting better. I will close this issue
